pkg/postage/batchstore/store.go

```diff
@@ -39,11 +39,10 @@ type store struct {
 	mtx sync.RWMutex
```

Comment:
It would be good to have a comment on what this mutex should guard and perhaps place it near a group of these fields.